pom.xml

```diff
@@ -410,12 +416,13 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
-        <version>2.10.3</version>
+        <version>3.2.0</version>
         <configuration>
           <source>${project.build.targetJdk}</source>
           <encoding>${project.build.sourceEncoding}</encoding>
           <maxmemory>${project.build.jvmsize}</maxmemory>
           <additionalparam>${javadoc.opts}</additionalparam>
+          <sourcepath>${project.basedir}/src/main;${project.basedir}/target/generated-sources/annotations</sourcepath>
```

Comment:
Need to include classes generated by Immutables

Reply:
Ah, Java 8 gives a warning, but Java 11 gives a fatal error